lib/opentracing/tracer.rb

```diff
@@ -1,19 +1,25 @@
 module OpenTracing
   class Tracer
-    # Start a new span
-    # @param operation_name [String] The name of the operation represented by the span
-    # @param child_of [Span] A span to be used as the ChildOf reference
-    # @param start_time [Time] the start time of the span
-    # @param tags [Hash] Starting tags for the span
+    # TODO(bhs): Support FollowsFrom and multiple references
+
+    # Starts a new span.
+    #
+    # @param operation_name [String] The operation name for the Span
+    # @param child_of [SpanContext] SpanContext that acts as a parent to
+    #        the newly-started Span. If a Span instance is provided, its
+    #        .span_context is automatically substituted.
```

Comment:
Has opentracing-ruby ever supported passing in a `span` instead of a `span_context`? If not, I think it shouldn't start supporting it cause the implicit conversion will confuse people sooner or later.